manifests/plugin/abrt/params.pp

```diff
@@ -1,6 +1,7 @@
 # Default parameters for the Pulp smart proxy plugin
 class foreman_proxy::plugin::abrt::params {
   $enabled                 = true
+  $version                 = 'installed'
```

Comment:
I'm not sure there is a point in keering ```abrt::params::version```. Maybe kafo needs it? @domcleal any reason we have group in params as well?